src/transformers/configuration_fsmt.py

```diff
@@ -0,0 +1,211 @@
+# coding=utf-8
+# Copyright 2019-present, Facebook, Inc and the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" FSMT configuration """
+
+
+import copy
+import logging
+
+from .configuration_utils import PretrainedConfig
+from .file_utils import add_start_docstrings_to_callable
+
+
+logger = logging.getLogger(__name__)
+
+FSMT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "stas/wmt19-ru-en": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-ru-en/config.json",
+    "stas/wmt19-en-ru": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-en-ru/config.json",
+    "stas/wmt19-de-en": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-de-en/config.json",
+    "stas/wmt19-en-de": "https://s3.amazonaws.com/models.huggingface.co/bert/stas/wmt19-en-de/config.json",
+}
+
+
+FSMT_CONFIG_ARGS_DOC = r"""
+    Args:
+        src_vocab_size (:obj:`int`, optional, defaults to None):
```

Comment:
Can we have all optional in backquotes (for italics), and no "defaults to None"? I know the other config files are not super good examples, but let's make the new ones better :-).

Reply:
Done!

We need to do it in one swoop, otherwise we always end up copying non-standard things and these need to be fixed. Let's make a PR where we nail all those ones in one change. If you're up for it please identify all the incomplient patterns and I will concoct some perl one liners to nails those.

Reply:
(WAY OFF TOPIC)
Here is my much worse alias for repo wide find and replace: 
```bash
git-replace () {
	git commit -m "Before replace"
	git grep -l "$1" | xargs sed -i '' -e "s/$1/$2/g"
	git diff --shortstat
}
```

Reply:
A neat one! It is excellent at that it skips any other files you may have in that dir, e.g. data files. I like it. Thank you!

The only drawback is that yours will miss files that aren't yet under git, which should be rare.

Here is mine:
```
find . -type d -name ".git" -prune -o -type f -exec perl -pi -e 's|OLDSTR|NEWSTR|g' {} \;
```
It safely skips files inside `.git`, but will replace files that aren't git - which may or may not be a good thing (e.g. bad for data files).